WorkloadManagementSystem/Utilities/PilotWrapper.py

```diff
@@ -21,6 +21,8 @@
 /usr/bin/env python << EOF
 
 # imports
+from __future__ import print_function
```

Comment:
this could also go to python 2.6 no ?

Reply:
This one is understood by python 2.6: https://docs.python.org/2/reference/simple_stmts.html#future-statements